lib/load.js

```diff
@@ -24,6 +24,14 @@ function load(target) {
     }
     // Load external IGs (e.g., US Core)
     recursiveLoadIGPath(`${__dirname}/definitions/${target}/IGs`, result);
+    // Load any other external IGs that have been specified in config
+    if (configuration.implementationGuide != null && configuration.implementationGuide.dependencies != null) {
+      for (const dep of configuration.implementationGuide.dependencies) {
+        if (fs.existsSync(dep.fileLocation)) {
```

Comment:
This requires the path to be absolute or relative to the working directory.  We don't want to require absolute paths because they don't work across systems (e.g., if you check in the config file, it might not work on someone else's system).

Relative paths, however, should be relative to the spec folder -- which isn't necessarily the same as the working directory.  We already support relative paths for the `examples` and `extraResources` of the config, so you should check to see how they do it and do it the same way.

Reply:
Makes sense, thanks for the note to check out `examples`. Fixed it here https://github.com/standardhealth/shr-fhir-export/pull/166/commits/fb40f694fe3e123d87ba5062a0052b55367a86f3.